### DISCUSSION TITLE
locked drops bypass

### DISCUSSION THREAD

**Discussion by LogGits** [GriefPrevention/GriefPrevention#960] [Ideas]
**As** a staff member
**I want** be able to bypass locked drops
**Because** I want to be able to rescue the items of a player who has gone afk.

Optional questions:

**Is your feature request related to a problem? Please describe.**
It's frustrating to see a player's drops and not be able to pick it up as a staff member.

**Describe alternatives you've considered**
I tried /ignoreclaims but that didn't bypass the drop protection.

**Comment by RoboMWM** [comment]
Unless otherwise decided to be in core, I'm leaning on not keeping drop locking as part of core (as this is an augmentation to a PvE-style server).
